### PR TITLE
XW-4456 | split CSS into 2 files

### PR DESCRIPTION
**Pull request by hakubo**

### CODE REVIEW

**Comment by hakubo** [comment]
https://github.com/filamentgroup/loadCSS/tree/v2.0.1 - preload is not supported everywhere, we should include polyfil after making sure this gives us any perf boost

File: app/inline-scripts/lazy-css.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+document.addEventListener('DOMContentLoaded', function () {
+	setTimeout(function () {
```
Comment (latata) [inline app/inline-scripts/lazy-css.js]: maybe we should load it even later - on ember startup?

File: app/components/fastboot-only/body-bottom.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import Component from '@ember/component';
-import {bool} from '@ember/object/computed';
+import {bool, equal} from '@ember/object/computed';
 import config from '../../config/environment';
 
 export default Component.extend({
 	tagName: '',
+	wikiVariable: null,
```
Comment (latata) [inline app/components/fastboot-only/body-bottom.js]: `wikiVariables` missing `s`

File: app/routes/application.js [hunk 3]
```diff
@@ -367,6 +367,7 @@ export default Route.extend(
 			headBottomComponent.appendTo(document.head);
 
 			bodyBottomComponent.set('queryParams', queryParams);
+			bodyBottomComponent.set('wikiVariable', wikiVariables);
```
Comment (latata) [inline app/routes/application.js]: missing `s`

File: app/styles/lazy.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
...%2C%20-12.000000%29%22%3E%3Cg%20transform%3D%22translate%2818.000000%2C%2012.000000%29%22%3E%3Crect%20x%3D%220%22%20y%3D%220%22%20width%3D%2224%22%20height%3D%2224%22%3E%3C%2Frect%3E%3Cpath%20d%3D%22M22%2C12.000125%20C22%2C11.4481336%2021.5936133%2C11.0001406%2021.0908574%2C11.0001406%20L5.10267629%2C11.0001406%20L9.91476788%2C5.70722332%20C10.2702426%2C5.31622943%2010.2702426%2C4.68423931%209.91476788%2C4.29324542%20C9.55929314%2C3.90225153%208.98471504%2C3.90225153%208.6292403%2C4.29324542%20L2.26615149%2C11.2921361%20C2.18160123%2C11.3851346%202.11523382%2C11.4951329%202.06886755%2C11.618131%20C1.97704415%2C11.8621272%201.97704415%2C12.1381228%202.06886755%2C12.382119%20C2.11523382%2C12.5051171%202.18160123%2C12.6151154%202.26615149%2C12.7081139%20L8.6292403%2C19.7070046%20C8.8065231%2C19.9020015%209.03926359%2C20%209.27200409%2C20%20C9.50474459%2C20%209.73748508%2C19.9020015%209.91476788%2C19.7070046%20C10.2702426%2C19.3160107%2010.2702426%2C18.6840206%209.91476788%2C18.2930267%20L5.10267629%2C13.0001094%20L21.0908574%2C13.0001094%20C21.5936133%2C13.0001094%2022%2C12.5521164%2022%2C12.000125%22%20fill%3D%22%23FFFFFF%22%3E%3C%2Fpath%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E);
```
Comment (latata) [inline app/styles/lazy.scss]: you can import whole `embedded_svg.scss` as it only contains variables

File: app/inline-scripts/lazy-css.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+		document.getElementById('lazy-css').media = 'all';
+	}, 500);
```
Comment (bkoval) [inline app/inline-scripts/lazy-css.js]: Could you add a comment why do we need to wait around 500 miliseconds?

**Review by bkoval — APPROVED** [review]
Code LGTM, just one request for a comment

**Review by latata — APPROVED** [review]

**Pull request merged**